README.aspnetcore-build.md

```diff
@@ -12,12 +12,7 @@ The [.NET Core Docker samples](https://github.com/dotnet/dotnet-docker/blob/mast
 
 # Linux amd64 tags
 
-- [`1.1.11-1.1.12-jessie`, `1.1.11-1.1.12`, `1.1`, `1` (*1.1/jessie/sdk/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1/jessie/sdk/Dockerfile)
```

Comment:
I see at least one reference in this readme to Windows Containers later on that should be cleaned up.

```
(Windows containers) Start a container from your image, get its assigned IP address, and then open your browser to the IP address of the container on port 80. To see console output, attach to the running container or use docker logs.
```